lib/importproject.h

```diff
@@ -153,6 +154,8 @@ namespace CppcheckXml {
     const char MaxCtuDepthElementName[] = "max-ctu-depth";
     const char CheckUnknownFunctionReturn[] = "check-unknown-function-return-values";
     const char Name[] = "name";
+    const char VSConfigurationElementName[] = "vsconfigurations";
```

Comment:
The variable names are fine. But the xml elements normally use "-" to separate names. So maybe "vs-configurations" and "config" are good names.